packages/discord-reporter/src/index.ts

```diff
@@ -31,19 +32,28 @@ const testStatusToEmoji = {
   timedOut: '❌',
   skipped: '⏸️',
   interrupted: '❌',
+  flaky: '🎲',
 };
 
 const GREEN = 47872;
 const RED = 13959168;
 
 const resultToStatus = {
-  passed: { color: GREEN, title: '🎉 Testing Completed!' },
-  failed: { color: RED, title: `${testStatusToEmoji.failed} Testing Failed!` },
+  passed: { content: undefined, color: GREEN, title: '🎉 Testing Completed!' },
+  failed: {
+    content:
+      process.env.DISCORD_DUTY_TAG &&
```

Comment:
wdyt to pass `process.env.DISCORD_DUTY_TAG ` this env as  `enabled` param - 
```
  discordReport: [
    '@lidofinance/discord-reporter',
    {
      enabled: process.env.DISCORD_REPORT_ENABLED,
    },
  ],
```

Reply:
possible)